krynn/krynn.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Name: Dragonlance Krynn locales
+ * Description: Set a random locale from the Dragonlance Realm of Krynn when posting. Based on the planets frindica addon by Mike Macgirvin and Tony Baldwin
+ * Version: 1.0
+ * Planets Author: Mike Macgirvin <http://macgirvin.com/profile/mike>
+ * Planets Author: Tony Baldwin <https://free-haven.org/profile/tony>
+ * Krynn modifications: Dylan Thiedeke
```

Comment:
ment "Author: ..." with the addition "Krynn" the author tag wont be identified and displayed. And you got a typo in it ;-)